fourcc.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import sys
+
+fourcc_include = sys.argv[1]
+
+ident_list = []
+with open(fourcc_include) as f:
+	for l in f.readlines():
+		parts = l.split()
+		if len(parts) < 3 or parts[0] != "#define":
+			continue
+		ident = parts[1]
+		if not ident.startswith("DRM_FORMAT_") and not ident.startswith("I915_FORMAT_"):
+			continue
+		if "(" in ident:
+			# Some complicated formats use a macro with parameters
+			continue
+		ident_list.append(ident)
+
+f_name = sys.argv[2]
+
+with open(f_name, "w+") as f:
```

Comment:
This just needs `w` instead of `w+`.
Also, can you consistently use the same type of quotes for all strings instead of mixing `'` and `"`.